source/Octostache/Templates/Functions/TextManipulationFunction.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace Octostache.Templates.Functions
+{
+    internal class TextManipulationFunction
+    {
+        public static string ToBase64(string argument, string[] options)
+        {
+            if (options.Any() || argument == null)
+            {
+                return null;
+            }
+            var argumentBytes = Encoding.UTF8.GetBytes(argument);
+            return Convert.ToBase64String(argumentBytes);
```

Comment:
Should we support `Base64FormattingOptions.InsertLineBreaks` through an option?

Reply:
dont mind leaving this one off as much, but if it can be done nicely through an option perhaps

Reply:
I am pretty sure Base64FormattingOptions isn't supported by dotnet standard, at least not standard 1.3...